base_layer/wallet_ffi/mobile_build.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
```

Comment:
It would be awesome if you gave some docs on what this script does, what dependencies you need installed to get it to run (NDK etc, Android SDK) and any additional help

Reply:
This is on my todo list.

I am also thinking about writing an environment bootstrap script, that will pull down all the tooling and repositories, populate the build config, etc (where possible). So it becomes more of a turnkey process for a contributor.